src/main/kotlin/com/ampnet/userservice/persistence/StringCryptoConverter.kt

```diff
@@ -0,0 +1,77 @@
+package com.ampnet.userservice.persistence
+
+import com.ampnet.userservice.exception.EncryptionException
+import com.ampnet.userservice.exception.ErrorCode
+import org.springframework.beans.factory.annotation.Value
+import java.security.GeneralSecurityException
+import java.security.InvalidAlgorithmParameterException
+import java.security.InvalidKeyException
+import java.security.NoSuchAlgorithmException
+import java.util.Base64
+import javax.crypto.BadPaddingException
+import javax.crypto.Cipher
+import javax.crypto.IllegalBlockSizeException
+import javax.crypto.NoSuchPaddingException
+import javax.persistence.AttributeConverter
+import javax.persistence.Converter
+
+@Converter
+class StringCryptoConverter : AttributeConverter<String, String> {
+
+    private val cipherInitializer = CipherInitializer()
+
+    @Value("\${spring.flyway.placeholders.private_key}")
+    private lateinit var databaseEncryptionKey: String
+
+    @Throws(
+        NoSuchAlgorithmException::class,
+        InvalidKeyException::class,
+        InvalidAlgorithmParameterException::class,
+        BadPaddingException::class,
+        NoSuchPaddingException::class,
+        IllegalBlockSizeException::class
+    )
+    override fun convertToDatabaseColumn(attribute: String?): String? {
+        if (attribute.isNullOrBlank()) return attribute
+        return try {
+            val cipher = cipherInitializer.prepareAndInitCipher(Cipher.ENCRYPT_MODE, databaseEncryptionKey)
+            encrypt(cipher, attribute)
```

Comment:
This is only possible `return` so no need to return from `try`.
```suggestion
            return encrypt(cipher, attribute)
```